### ISSUE TITLE
Auto-refresh feature

### ISSUE THREAD

**Issue by jeanmonod** [liip/TheA11yMachine#19]
It would be great to have an auto refresh functionally available in the HTML report.

**Comment by Hywan** [comment]
Why?

**Comment by jeanmonod** [comment]
Yes, but I will not automatically activate it. Maybe a nice switcher in the top right?

**Comment by Hywan** [comment]
Why a switcher? Is it something someone would like to disable?

**Comment by jeanmonod** [comment]
It depends the workflow that you are using. Maybe if we have #22 then it's no more useful. But a workflow I can imagine, is (as a developper):
* I run a full audit of a website
* I open the report in my browser
* I activate the manual auto refresh
* I fix the issue page per page, continuously running audit for a single page
* With #8 implemented I see my global report getting better and better

**Comment by Hywan** [comment]
Yup, sounds good!

**Comment by krtek4** [comment]
Eventually with a progress bar ?

**Comment by krtek4** [comment]
I think we have two different things here :

# Auto-refresh of the index page while the crawler is still running to see new URLs
# Continuous auditing of the URLs to display improvements (which can be done for the whole report or on a per URL basis).
